test/test-addon.js

```diff
@@ -52,6 +64,23 @@ test('build simple addon', function (t) {
   proc.stderr.setEncoding('utf-8')
 })
 
+test('make sure addon symbols do not overlap', function (t) {
+  t.plan(3)
+
+  var addonPath = path.resolve(__dirname, 'node_modules', 'duplicate_symbols')
+  // Set the loglevel otherwise the output disappears when run via 'npm test'
+  var cmd = [nodeGyp, 'rebuild', '-C', addonPath, '--loglevel=verbose']
+  var proc = execFile(process.execPath, cmd, function (err, stdout, stderr) {
+    var logLines = stderr.toString().trim().split(/\r?\n/)
```

Comment:
Superfluous toString(), it's already a string. (I realize you copied it but still.)